services/ui-src/src/components/fields/DynamicField.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect } from "react";
+import { useFieldArray, useFormContext } from "react-hook-form";
+// components
+import { Box, Button, Flex, Image } from "@chakra-ui/react";
+// utils
+import { makeMediaQueryClasses } from "utils";
+// assets
+import cancelIcon from "assets/icons/icon_cancel_x_circle.png";
+import { TextField } from "./TextField";
+
+export const DynamicField = ({ name, label }: Props) => {
+  const mqClasses = makeMediaQueryClasses();
```

Comment:
Don't really need this here -- actually should probably remove it from any components where we aren't directly using .desktop or .tablet or .mobile.
```suggestion
```